src/ceph-disk

```diff
@@ -1719,6 +1804,9 @@ def main_prepare(args):
 
         if args.zap_disk is not None:
             zap(args.data)
```

Comment:
The reason I wrote the patch in the first place, was that zeroed disks journals caused issues with 0.80 and 0.94 for journals, and also caused issues on 0.80 for data. Since this is no longer an issue for 0.94 I did not forward port this solution.